### PR TITLE
PREF-247 | UseCase for defaulting record key to property name

### PR DESCRIPTION
**Pull request by jpmarcotte**
Relevant snippet from `UsesDefaultInterface`

```php
public const PROP_SAME_AS_DEFAULT = 'same_as_default';
public const PROP_USES_DEFAULT = 'uses_default';
public const PROP_DIFFERENT_THAN_DEFAULT = 'something_else';
```

### CODE REVIEW

File: UseCase72/composer.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    "prefab"
+  ],
+  "config": {
+    "sort-packages": true,
+    "platform": {
+      "php": "7.3"
+    }
+  },
+  "require": {
+    "neighborhoods/prefab": "dev-PREF-247-7x-default-record-key-to-name"
```
Comment (jpmarcotte) [inline UseCase72/composer.json:15]: TODO: Change to a release tag after the relevant PR is merged and released.

File: UseCase72/README.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# Use Case 71
+Testing whether or not the Deprecated flag works
```
Comment (arielallon) [inline UseCase72/README.md:2]:
```suggestion
# Use Case 72
Testing fallback to property name when record_key is not specified
```

File: UseCase72/composer.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "neighborhoods/prefab-fitness-use-case-51",
```
Comment (arielallon) [inline UseCase72/composer.json:2]:
```suggestion
{
  "name": "neighborhoods/prefab-fitness-use-case-72",
```

**Review by arielallon — CHANGES_REQUESTED** [review]
just a couple of minor housekeeping changes requested

**Review by mucha55 — APPROVED** [review]

**Review by mucha55 — APPROVED** [review]

**Review by arielallon — APPROVED** [review]

**Pull request merged**